Package.swift

```diff
@@ -12,7 +12,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/hummingbird-project/hummingbird.git", from: "2.0.0-rc.1"),
-        .package(url: "https://github.com/hummingbird-project/swift-jobs.git", from: "1.0.0-beta.1"),
+        .package(url: "https://github.com/thoven87/swift-jobs.git", branch: "stevenson/queued-at"),
```

Comment:
TODO: revert after swift Swift Jobs [PR](https://github.com/hummingbird-project/swift-jobs/pull/11) is merged